src/bindings/io/uv/README.md

```diff
@@ -0,0 +1 @@
+# IO-Binding for libuv
 
 ## Requirements
 
 - libuv (version 1.x is
recommended)
 
 ## Usage
 
 Use the `elektraIoUvNew` function to get a new
binding.
 
 Make sure to build your application with `libelektra-io-uv`
and `libuv` or simply use `pkg-config --cflags --libs libelektra-io-uv`.

 ### ElektraIoInterface * elektraIoUvNew (uv_loop_t * loop)
 Create
and initialize a new IO binding.
 
 *Parameters*
 - loop Loop to use for
IO operations
 
 *Returns* Populated IO interface
 
 ## Example
 
```C
#include <elektra/io_uv.h>

void main (void)
{
	KDB* repo;

	// ... open KDB

	// Create libuv event loop
	uv_loop_t * loop = uv_default_loop ();

	// Initialize I/O binding tied to event loop
	ElektraIoInterface * binding = elektraIoUvNew (loop);

	// Initialize notification wrapper
	elektraNotificationOpen (kdb, binding);

	// Start the event loop
	uv_run (loop, UV_RUN_DEFAULT);

	// Cleanup
	binding->cleanup (binding);
	uv_loop_close (loop);
	free (loop);
}
```

```

Comment:
Line breaks broken? dos2unix?

Reply:
Missed that when commiting from Windows.